pkg/auth/tokens/manager.go

```diff
@@ -810,10 +811,11 @@ func ParseTokenTTL(ttl string) (time.Duration, error) {
 	return dur, nil
 }
 
+// ValidateMaxTTL will return the duration of the provided TTL or the duration of the max allowed TTL whichever is smaller.
 func ValidateMaxTTL(ttl time.Duration) (time.Duration, error) {
```

Comment:
I think we should name this since validate implies an error would be returned if it exceeds the MaxTTL setting. May it can be renamed to `ClampWithMaxTTL` or anything else that is more accurate.